cc_utilities/command_line/sync_redcap_to_commcare.py

```diff
@@ -107,6 +110,14 @@ def main_with_args(
                 .pipe(normalize_phone_cols, phone_cols)
                 .pipe(collapse_housing_fields)
                 .pipe(set_external_id_column, external_id_col)
+                .pipe(
```

Comment:
Do you think `export_records` (above) should include `filter_logic` (https://github.com/redcap-tools/PyCap/blob/master/redcap/project.py#L335) to exclude rows that have been previously rejected? My gut says  it may be confusing if we still attempt to sync these without some manually marketing them as ready to sync again (perhaps by learning or updating `integration_status`)...

Reply:
I hadn't even considered whether REDCap would keep trying to send these records. 

Considering that, I agree it's a good idea to filter them and skip records that were already rejected. 

I do think we'd have to collaborate with the philly team and figure out what kind of workflow they would want for reviewing rejected records and re-syncing them. Right now the integration_status column is set up to only accept two values; 'person_rejected' and 'sent_to_commcare' (was mentioned [here](https://caktusgroup.slack.com/archives/C016RMY7YAC/p1621885132002800?thread_ts=1621882905.000800&cid=C016RMY7YAC). 
I imagine there could be a third value added like 'rejection_reviewed' or another column.
I will ask Sherjeel about this tomorrow morning!

Reply:
(For the record)
In our last check-in call, we learned that the Philly team is using a separate variable for tracking the review status, and so we can accurately filter out cases with an integration_status value. 

I added filtering in https://github.com/caktus/commcare-utilities/pull/64/commits/3dfd06141f7bee391152f2bbc71c7ca8ca962436